src/app/contribute/overview/get-code/index.html

```diff
@@ -0,0 +1,27 @@
+<stache
+  pageTitle="Get the code"
+  showTableOfContents="true"
+  layout="sidebar">
+
+  <stache-page-summary>
+    <p>
+      To contribute to SKY UX, fork <a stacheRouterLink="/learn/get-started/prereqs">the appropriate repo</a> and create a kebab-case branch to commit and push changes.
+    </p>
+  </stache-page-summary>
+
+  <ol>
+    <li>
+      <a href="https://help.github.com/articles/fork-a-repo/">Fork the repo</a> into your own GitHub repo. For a list of the SKY UX repositories that target different areas of development, see the <a stacheRouterLink="/contribute/packages">packages matrix</a>.
+    </li>
+    <li>
+      Create a branch off of <sky-code>master</sky-code>. Name it after the feature to contribute, and use kebab-case (<sky-code>my-new-feature</sky-code>).
+    </li>
+    <li>
+      Clone your repo locally and run <sky-code>nvm use</sky-code> to ensure you have the preferred version of Node.js installed. Then run <sky-code>npm install</sky-code> to install all required dependencies. For more information on prerequisites, see <a stacheRouterLink="/learn/get-started/prereqs">SKY UX prerequisites</a>.
```

Comment:
@Blackbaud-BobbyEarl, what do you think? Should we switch `npm install` to `skyux install` here? I assume so since I think these docs predate `skyux install`, but I just want to make sure.

Reply:
In this particular case, `npm install` is still the correct instruction.

Reply:
OK. Thanks! I'll leave it as is.